scripts/data_deletion/delete_training_data_utils.js

```diff
@@ -207,7 +205,7 @@ var filterFamilyMembers = function(personsList, logdir) {
 
 // Find which facilities (if any) this person is a contact for.
 var isContactFor = function(db, personId) {
-  return db.query('medic-client/places_by_contact', {key: [personId], include_docs: true})
```

Comment:
I don't follow. It goes in medic if it's only needed on the server, and medic-client if it's needed on the client. It gets moved if it doesn't (or no longer) follows that rule.